libraries/lib-audio-io/AudioIO.cpp

```diff
@@ -1139,11 +1152,18 @@ bool AudioIO::AllocateBuffers(
       try
       {
          if( mNumPlaybackChannels > 0 ) {
-            // Allocate output buffers.  For every output track we allocate
-            // a ring buffer of ten seconds
+            // Allocate output buffers.
+            // Allow at least 2x of the buffer latency.
             auto playbackBufferSize =
-               (size_t)lrint(mRate * mPlaybackRingBufferSecs.count());
+               std::max((size_t)lrint(mRate * mPlaybackRingBufferSecs.count()), mHardwarePlaybackLatencyFrames * 2);
+
+            // Make playbackBufferSize a multiple of mPlaybackSamplesToCopy
+            playbackBufferSize = mPlaybackSamplesToCopy *
+               ((playbackBufferSize + mPlaybackSamplesToCopy - 1) / mPlaybackSamplesToCopy);
 
+            // Adjust mPlaybackRingBufferSecs correspondingly
+            mPlaybackRingBufferSecs = PlaybackPolicy::Duration { playbackBufferSize / mRate };
```

Comment:
Irrelevant to the bug fix, but I notice now that `mPlaybackRingBufferSecs` can be a local variable of this function only, eliminating a member variable